OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -119,6 +151,22 @@
     </codeblock>
 </div>
 
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="CTL-x100 Android Mode">
+    Wacom CTL-x100 Android Mode
+</h3>
+<div class="ms-3">
+<p>
+    Sometimes the Ctl-x100s can be left in Android mode instead of PC mode, to fix this press
+    the outer 2 express keys for 3-4 seconds until they blink, this will change the mode between
+    PC and Android.
```

Comment:
```suggestion
    CTL-x100 tablets boots in Android mode instead of PC mode in some rare cases. To fix this, press
    the outer 2 express keys for 3-4 seconds until the LEDs change brightness. This toggles the tablet's operating mode
    between PC (high LED brightness) and Android mode (low LED brightness).
```

https://support.wacom.com/hc/en-us/articles/1500006332582-How-can-I-manually-switch-the-Wacom-Intuos-into-Android-mode-